src/components/editions/teamScore/index.tsx

```diff
@@ -86,27 +87,30 @@ const scorerStyles = (location: TeamLocation): SerializedStyles => css`
 	}
 `;
 
-const TeamScore: FC<Props> = ({ team, location }) => (
-	<section css={styles(location)}>
-		<div css={scoreStyles(location)}>
-			<div css={scoreNumberStyles}>
-				<span css={scoreInlineStyles}>{team.score}</span>
+const TeamScore: FC<Props> = ({ team, location }) => {
+	const scorers = fromNullable(team.scorers);
```

Comment:
Interesting. The type says this _can't_ be missing. Is it possible the type is lying to you because of the type casts (`foo as bar`) in `footballContent.ts`?

Reply:
I feel like we've had this discussion before - sometimes the scorers array can be empty so the page will hang if we try to map ie:
```{team.scorers.map((scorer) => (
							<li key={`${scorer.player}`}>
								{scorer.player} {scorer.timeInMinutes}&apos;{' '}
								{scorer.additionalInfo}
							</li>
						))}
						```

Reply:
this works though:
```
{team.scorers && (
<ul css={scorerStyles(location)}>
    {team.scorers.map((scorer) => (
	    <li key={`${scorer.player}`}>
		    {scorer.player} {scorer.timeInMinutes}&apos;{' '}
		    {scorer.additionalInfo}
	    </li>
    ))}
    </ul>
)}

Reply:
And now i remember the original reason to add `fromNullable`:
```
  107:6  error  Unnecessary conditional, value is always truthy  @typescript-eslint/no-unnecessary-condition